payment/service.go

```diff
@@ -31,11 +212,31 @@ func InitService(datastore Datastore) (*Service, error) {
 		return nil, err
 	}
 
+	// create a background context with cancellation
+	ctx, cancel := context.WithCancel(context.Background())
+
 	service := &Service{
 		wallet:    *walletService,
 		cbClient:  cbClient,
 		datastore: datastore,
+		cancel:    cancel,
+	}
+	err = service.InitKafka()
+	if err != nil {
+		return nil, err
+	}
+
+	// init vote drain worker
+	err = service.InitKafka()
+	if err != nil {
+		return nil, err
 	}
+
+	// kick off async goroutine to monitor the vote
+	// queue of uncommitted votes in postgres, and
+	// push the votes through redemption and kafka
+	go service.DrainVoteQueue(ctx, 15*time.Second)
```

Comment:
it starts a process in parallel. basically go do this and i'll continue on

Reply:
No, it will not be kicked off again on failure or panic within the goroutine.  There is no restart functionality, but this could be added by having the worker periodically (maybe by watching for a different `<-time.After` report back over a live-ness channel that is passed into DrainVoteQueue or something.  I would say for the time being this might be overkill.  We have logging in place in the event to loop completes.

Yes, this is kicking off a worker event loop in a new goroutine.  It should keep running until the context is cancelled.